lucille-core/src/main/java/com/kmwllc/lucille/stage/DictionaryLookup.java

```diff
@@ -34,12 +35,15 @@
  *       for a 1-1 mapping of results or supply one destination field for all of the source fields to be mapped into.
  * <p>  - dict_path (String) : The path the dictionary to use for matching. If the dict_path begins with "classpath:" the classpath
  *       will be searched for the file. Otherwise, the local file system will be searched.
- * <p>  - use_payloads (Boolean, Optional) : denotes whether paylaods from the dictionary should be used or not. Defaults to true.
+ * <p>  - use_payloads (Boolean, Optional) : denotes whether payloads from the dictionary should be used or not. Defaults to true.
  * <p>  - update_mode (String, Optional) : Determines how writing will be handling if the destination field is already populated.
  *      Can be 'overwrite', 'append' or 'skip'. Defaults to 'overwrite'.
  * <p>  - set_only (Boolean, Optional) : If true, the destination field will be set to true if all values in the source field
  *      are present in the dictionary.
- * <p>  - ignore_missing_source (Boolean, Optional) : Intended to be used in combination with set_only. If true, the destination field
+ * <p>  - use_any_match (Boolean, Optional) : Use in combination with set_only. If true, the destination field will
```

Comment:
We should improve the documentation around set_only to be more specific about what it is doing and/or include references to the use_any_match/ignore_missing_source params. ie:

```
- set_only (Boolean, Optional) : If true, will set the dest field to `true` when a match is found, instead of outputting the match itself. Specific behavior can be controlled via `use_any_match` and `ignore_missing_source`. Defaults to false.
```